app/controllers/slides_controller.rb

```diff
@@ -165,6 +165,11 @@ def embedded
       @slide.increment(:embedded_view).increment(:total_view).save
     end
     @start_position = slide_position
+    if params.has_key?(:inside) && params[:inside] == '1'
+      @is_base62 = false
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name '@is_base62' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)